CHANGELOG.md

```diff
@@ -0,0 +1,274 @@
+
+
+### Installer Binaries
+| filename | Size | sha512 hash |
+| -------- | ---- | ----------- |
+| [keadm-v0.3.0-beta.0-linux-amd64.tar.gz](https://github.com/kubeedge/kubeedge/releases/download/v0.3.0-beta.0/keadm-v0.3.0-beta.0-linux-amd64.tar.gz) | 2.05 MB | `5775c5c1b25a9148fd592cf61e0cd671ca8fb345c63c897709bda2c479bb3a315c9f979e934fbd5a79dfcb633f793ffe948979773e65f95c65791b4ac08c3d0f` |
+
+## Changelog since v0.2.1
+
+### Features Added
+
+
+- Device Management using CRDs : This feature mainly provides APIs for managing devices from cloud and synchronize the device updates between cloud and edge. DeviceController is the module in cloud which is responsible for watching on devices and devicemodels and syncing the data to edge. DeviceController is also responsible for receiving the device-twin updates sent from edge and making the changes in cloud(apiserver). DeviceController also creates a configmap for each node and adds the deviceInstances, protocols, properties and propertyVisitors in it. This configmap can be consumed by any application like mapper and used to configure/control devices.
+
+
+- Bluetooth Mapper : Bluetooth Mapper is an application that is used to connect and control devices that use bluetooth as a communication protocol. It consumes the configmap generated by device-controller and reuires a configuration file for information not present in configmap. Configurations can be changed at runtime by providing the input through the MQTT broker. It can run schedule based actions and converts data received from devices into edge_core expected format.
+
+
+- Performance Test Framework : Benchmarking framework to measure the performance against the following Service Level Objectives:
+    - Latency: time cost from the moment when the server gets the request to last byte of response sent to the users.
+    - Throughput: measure how many requests can be served within given time.
+    - Scalability: potential scaling capacity (including number of nodes, pods, devices etc. at the edge) under different load conditions.
+    - CPU Usage: measure the cpu usage of KubeEdge under different load conditions.
+    - Memory Usage: measure the memory usage of KubeEdge under different load conditions.
+
+
+- KubeEdge Installer : Support for basic commands to bootstrap and teardown both KubeEdge cloud and edge components.
+Supported Commands :
+    - `kubeedge init` : bootstrap cloud prerequisites and cloud components if not already present.
+    - `kubeedge join` : add edge node in cloud api-server, bootstrap edge prerequisites and edge components if not already present.
+    - `kubeedge reset` : for cloud node teardown cloud components and prerequites. For edge node teardown edge components and prerequisites.
```

Comment:
see previous comment for repetitive content.

Reply:
Done, Thanks

Reply:
Done, Thanks